src/components/RelatedArticles.jsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+} from "@components/ui/carousel";
+
+export default function CarouselSpacing(props) {
+  return (
+    <section>
+      <h2 className="mx-auto text-center">Related Articles</h2>
+      <Carousel className="mx-auto w-full max-w-full p-4">
+        <CarouselContent className="-ml-1 p-4">
+          {props.firstArticle && props.length >= 1 && (
```

Comment:
I think you can use a `.map` because there's nothing static here

Reply:
Changed to map